wordpress-mobile/WordPress-Android
# Connect Blogging Prompts card to store

This PR connects Blogging Prompt Card and Store through My Site source management framework.  

Remove user Id from the train of faces Avatar.
Blogging Prompt Card source and tests are shamelessly based on the CardSource code.

Card is the main way we keep prompts in sync - as long as you visit My Site screen, it loads 20 prompts, starting from current date.

To test:
- Enable blogging prompt feature.
- Restart the app.
- Navigate to My Site fragment and open Home dashboard.
- Confirm that the Blogging Prompt card is visible and shows live prompt (I still need to fix the max lines value on the card).
- Close the app and forward the date on your device.
- Open the app and confirm that the blogging prompt card contains different prompt.

## Regression Notes
1. Potential unintended areas of impact
Card is visible when FF is OFF.
Card update logic is not working properly.

2. What I did to test those areas of impact (or what existing automated tests I relied on)
Manual and automatic testing.

3. What automated tests I added (or what prevented me from doing so)
BP source test.

PR submission checklist:
